config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path.yaml

```diff
@@ -1,24 +1,155 @@
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
+
+periodics:
+- interval: 6h
+  name: ci-kubernetes-csi-stable-on-kubernetes-master
+  decorate: true
+  extra_refs:
+  # TODO: replace with kubernetes-csi/csi-driver-host-path master once branch prow is merged
+  - org: pohly
+    repo: csi-driver-host-path
+    base_ref: prow
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  spec:
+    containers:
+    # We need this image because it has Docker in Docker and go.
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-master
+      command:
+      - runner.sh
+      args:
+      - ./.prow.sh
+      env:
+      # The current stable set of Kubernetes-CSI components is meant to be compatible
+      # with future Kubernetes, except for alpha features. Those therefore do not get
+      # tested.
+      - name: CSI_PROW_KUBERNETES_VERSION
+        value: "latest"
+      - name: CSI_PROW_BUILD_JOB
+        value: "false"
+      - name: CSI_PROW_DEPLOYMENT
+        value: "kubernetes-1.13"
+      - name: CSI_PROW_E2E_ALPHA_FOCUS
+        value: none
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+    resources:
+      requests:
+        # these are both a bit below peak usage during build
+        # this is mostly for building kubernetes
+        memory: "9000Mi"
+        # during the tests more like 3-20m is used
+        cpu: 2000m
+
+- interval: 6h
+  name: ci-kubernetes-csi-stable-on-kubernetes-1-13
+  decorate: true
+  extra_refs:
+  # TODO: replace with kubernetes-csi/csi-driver-host-path master once branch prow is merged
+  - org: pohly
+    repo: csi-driver-host-path
+    base_ref: prow
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  spec:
+    containers:
+    # We need this image because it has Docker in Docker and go.
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-master
+      command:
+      - runner.sh
+      args:
+      - ./.prow.sh
+      env:
+      # The current stable set of Kubernetes-CSI components is meant to be compatible
+      # with Kubernetes 1.13.x, including alpha features.
+      - name: CSI_PROW_KUBERNETES_VERSION
+        value: "release-1.13"
+      - name: CSI_PROW_BUILD_JOB
+        value: "false"
+      - name: CSI_PROW_DEPLOYMENT
+        value: "kubernetes-1.13"
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+    resources:
+      requests:
+        # these are both a bit below peak usage during build
+        # this is mostly for building kubernetes
+        memory: "9000Mi"
```

Comment:
tracking not needing so much resources here: https://github.com/kubernetes-sigs/kind/issues/381